about.html

```diff
@@ -0,0 +1,144 @@
+                    <p class="para1">
+                        "Hello! Sharing world"
+                    </p>
+                    <h1 class="main-title" data-text="COLLISION INDUSTRY CONFERENCE 2023">
+                        COLLISION INDUSTRY CONFERENCE 2023
+                    </h1>
+                    
+                    <p class="para2">
+                        Collision brings together the people and companies redefining the global tech industry.
+
+                        Web Summit events have gathered half a million people across Web Summit in Lisbon, Web Summit Rio in South America, Collision in North America, and RISE in Asia since its beginnings as a 150-person conference in Dublin in 2009.
+                        Web Summit’s mission has been to create software that enables meaningful connections between the CEOs, founders, investors, media, politicians and cultural figureheads who are reshaping the world.
+                    </p>
+                    <p>
+                        If you have any questions related to the Conference, you may email us on:
+                        <a href="#">
+                          <b>
+                            info@collisionconf.com <span>&GT;</span>
+                          </b>
+                        </a>
+                      </p>
+                </div>
+                <h2 class="about-title">
+                    COLLISION INDUSTRY <br> CONFERENCE 2023 <br> LOGO
+                </h2>
+                <hr class="line">
+                <p class="para2">
+                    Collision was created by the team behind Web Summit and is North America’s fastest-growing technology event.
+                </p>
+                <div class="logo">
+                    <img src="./IMAGES/collision-logo.avif" alt="my-logo">
+                    <div id="gallery"></div>
+                </div>
+                <h3 class="mini-about">
+                    See the past CC Global Summits
+                </h3>
+                <div class="images-about">
+                    <div class="img-about one">
+                        <h3>
+                            <big>2019</big>
+                        </h3>
+                    </div>
+                    <div class="img-about two">
+                        <h3>
+                            <big>2020</big>
+                        </h3>
+                    </div>
+                    <div class="img-about three">
+                        <h3>
+                            <big>2022</big>
+                        </h3>
+                    </div>
+                </div>
+            </section>
+        </article>
+    </main>
+
+    <footer>
+        <div class="foot-last">
+            <h2>
+              <span>CC</span><big>COLLISION  CONFERENCE</big>
+            </h2>
+            <h3>
+                Enercare Centre, Toronto<br> 
+                All Rights Reserved
+            </h3>
+          </div>         
+    </footer>
```

Comment:
- [ ] On the `About page` of your project, the `Partner` section is missing on `mobile view`. Kindly add it and you'll be sure to replicate the original design as much as possible.
   - Also kindly give the footer some `margin-left` or `padding-left` so that the design will not be hitting the wall to the left.


| Your Design 👇 | Behance design 👇  |
| ------------- | ------------- |
| ![rik01](https://user-images.githubusercontent.com/78634841/217528832-b637b384-f7ef-4373-b8e7-57613732232e.png) | ![image](https://user-images.githubusercontent.com/78634841/217528874-168d1ca5-3bfe-4de9-9b2f-bf1e03af2535.png) |